spec/rails_helper.rb

```diff
@@ -191,6 +191,10 @@ def read_csv_from_zip(zip_file, filename)
     TestSuiteTimeMachine.reset
   end
 
+  config.after do
```

Comment:
Could the failures be related to removing this?

https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/config/environments/test.rb#L1

Reply:
No, this is something else

Reply:
Hmm, I wonder why 🤔